src/lib/journey/callbacks/choice/choice.stories.js

```diff
@@ -21,3 +26,54 @@ export const Base = {
     callback: step.getCallbackOfType(CallbackType.ChoiceCallback),
   },
 };
+
+export const Radio = {
+  args: {
+    ...Base.args,
+    displayType: 'radio',
+  },
+};
+
+const Template = (args) => ({
+  Component: Input,
+  props: args,
+});
+
+export const Interaction = Template.bind({});
+
+Interaction.args = { ...Base.args };
+
+Interaction.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const cb = step.getCallbacksOfType(CallbackType.ChoiceCallback)[0];
+  const select = canvas.getByLabelText('Choose one');
+
+  await new Promise((resolve) => setTimeout(resolve, 1000));
+
+  await userEvent.selectOptions(select, '0');
+  await expect(cb.getInputValue()).toBe(0);
+
+  await new Promise((resolve) => setTimeout(resolve, 1000));
+
+  await userEvent.selectOptions(select, '1');
+  await expect(cb.getInputValue()).toBe(1);
+};
+
+export const RadioInteraction = Template.bind({});
+
+RadioInteraction.args = { ...Radio.args };
```

Comment:
In response to a previous conversation we had, I decided to add two interaction tests here - one for choice / select, and another for choice / radio. I think this is the only place we have multiple interactions for the same component - I'd welcome your thoughts on this approach generally, naming (i.e. Interaction vs RadioInteraction), and if there is a more concise way to achieve this.

Reply:
I think this is fine.